index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const app = express()
+const PORT = process.env.PORT || 3000
+
+const buildDir = `${__dirname}/build`
+
+app.use(
+  express.static(buildDir, { maxAge: 2629746, immutable: true, index: false })
```

Comment:
@sjedlikowski WDYT? For Static files (1 month)?

Reply:
Yup, 1 month will be fine as well